src/test/java/com/acrolinx/sidebar/utils/LoggingUtilsTest.java

```diff
@@ -26,7 +26,6 @@ public void setupLogging() throws Exception
     {
         LoggingUtils.setupLogging("TEST");
         String logFileLocation = LoggingUtils.getLogFileLocation();
-        Assert.assertTrue(logFileLocation != null);
```

Comment:
I can see that there is an instance when LoggingUtils.getLogFileLocation() returns null, so we have only tested for the case when it's not null.

Reply:
I remove this assertTrue since it did not add any value in my opinion. If logFileLocation is actually null, the test will fail anway further down in line 31 when passing that to the `new URI`.

In general, the whole design of this test class is questionable. It depends on logback, which is also a compile dependency of the whole sidebar-sdk-java. In the future this should probably be changed, to only depend on slf4j like a normal Java library should and let the user of the library choose and configure the logging framework. But since that would be a change which breaks compatibility with previous versions, we cannot do it right away but have to carefully plan if and when we do it (e.g. increase the major version number for that, provide some changelog documentation for users, etc.).

Reply:
This is well understood now